scala/private/common_attributes.bzl

```diff
@@ -135,3 +135,17 @@ resolve_deps = {
         allow_files = False,
     ),
 }
+
+test_resolve_deps = {
```

Comment:
Can't/Shouldn't this be confined to scala_test.bzl?

Reply:
Good point - I moved it blindly but you are right that it's only used by `scala_test`